src/sql/base/browser/ui/panel/panel.component.ts

```diff
@@ -95,6 +125,14 @@ export class PanelComponent implements AfterContentInit, OnInit {
 				this._activeTab = tab;
 				this.setMostRecentlyUsed(tab);
 				this._activeTab.active = true;
+
+				// Make the tab header focus on the new selected tab
+				let activeTabHeader = this._headerTabs.find(i => i.tab === this._activeTab);
+				if (activeTabHeader && activeTabHeader.actionHeaderRef) {
```

Comment:
I'd prefer if we just add a focus() function to the TabHeaderComponentClass, rather than directly accessing it's element from here.